ci-operator/step-registry/ibmcloud/provision/vpc/mult-subnets/ibmcloud-provision-vpc-mult-subnets-commands.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+extra_subnets_count=$((EXTRA_SUBNETS_NUMBER))
+#15 is the max number of of ALB, so just create more subnets on vpc, do not use them when create cluster. https://cloud.ibm.com/docs/vpc?topic=vpc-load-balancer-faqs#max-number-subnets-alb
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output   
+    region="${LEASED_RESOURCE}"
+    export region
+    "${IBMCLOUD_CLI}" config --check-version=false
+    echo "Try to login..." 
+    "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+function waitAvailable() {
+    local retries=15  try=0 
+    local type="$1" name="$2"
+    while [ "$(${IBMCLOUD_CLI} is ${type} ${name} --output JSON | jq -r '.status')" != "available" ] && [ $try -lt $retries ]; do
+        echo "The ${name} is not available, waiting..."
+        sleep 10
+        try=$(expr $try + 1)
+    done
+
+    if [ X"$try" == X"$retries" ]; then
+        echo "Fail to get available ${type} - ${name}"
+        "${IBMCLOUD_CLI}" is ${type} ${name} --output JSON
+        return 1
+    fi   
+}
+
+function createSubnet() {
+    local subnetPreName="$1" vpcName="$2" zone="$3" id="$4"
+    local subnetName pgwName
+    pgwName=$(ibmcloud is subnets | grep ${subnetPreName}-${zone} | awk '{print $7}')
+    subnetName="${subnetPreName}-${id}"
+    "${IBMCLOUD_CLI}" is subnet-create ${subnetName} ${vpcName} --pgw ${pgwName} --ipv4-address-count 16 --zone ${zone}
```

Comment:
when create extra subnet, I used **--ipv4-address-count** option, **--pgw** is required. 
in ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh, create the subnet, then attach_public_gateway_to_subnet.
    _"${IBMCLOUD_CLI}" is subnet-create ${subnetName} ${vpcName} --ipv4-cidr-block "${zone_cidr_main}/24"_